### DISCUSSION TITLE
下载视频的界面没有下载按钮

### DISCUSSION THREAD

**Discussion by Chanldon-Ntwo** [the1812/Bilibili-Evolved#3083] [提问 / Q&A] [answered]
请问点击下载视频后，弹出的下载窗口中没有显示下载按钮是什么原因呢？仅可以设置清晰度、格式等选项。
我的浏览器是Edge Chromium（99.0.1150.30），Tampermonkey版本是v4.13.6138，Bilibili Evolved是v2.1.5，均是目前发布的最新版。

**Comment by the1812** [comment, marked as answer]
#2718?

**Reply by Chanldon-Ntwo** [reply]
十分感谢，问题解决了！
